dev/services/wms/ows/wms_cfg.py

```diff
@@ -2164,6 +2165,368 @@
             },
         ]
     },
-
+    {
+        "name": "fcp_green_veg",
+        "title": "Fractional Cover Percentiles - Green Vegetation",
+        "abstract": "",
+        "products": [
+            {
+                "label": "Fractional Cover Percentiles - Green Vegetation",
+                "abstract": """
+Fractional Cover Percentiles version 2.0.0, 25 metre, 100km tile, Australian Albers Equal Area projection (EPSG:3577). Data is only visible at higher resolutions; when zoomed-out the available area will be displayed as a shaded region.
+Fractional cover provides information about the the proportions of green vegetation, non-green vegetation (including deciduous trees during autumn, dry grass, etc.), and bare areas for every 25m x 25m ground footprint. Fractional cover provides insight into how areas of dry vegetation and/or bare soil and green vegetation are changing over time.  The percentile summaries are designed to make it easier to analyse and interpret fractional cover.  For example the 90th percentile of bare soil for a particular year will identify areas that have experienced a high portion of bare soil during that year.  The fractional cover algorithm was developed by the Joint Remote Sensing Research Program.
+This contains a (10th, 50th and 90th percentile) for green vegetation observations acquired in each full calendar year (1st of January - 31st December) from 1987 to the most recent full calendar year.
+""",
+                "type": "100km tile",
+                "variant": "25m",
+                "name": "fcp_green_veg",
+                "product_name": "fcp_green_veg",
+                "min_zoom_factor": 15.0,
+                "zoomed_out_fill_colour": [150, 180, 200, 160],
+                "time_zone": 9,
+                "extent_mask_func": lambda data, band: data["extent"] == 1,
+                "ignore_info_flags": [],
+                "data_manual_merge": False,
+                "always_fetch_bands": ["extent"],
```

Comment:
No band `extent` in the FCP product
```suggestion
                "always_fetch_bands": [ ],
```